src/builders/cmake.ts

```diff
@@ -199,6 +199,33 @@ class CMakeBuilder implements IBuilder {
       return true;
     }
   }
+
+  /**
+   * Some errors have file:line:col encoded in the error log
+   * @param s error log
+   * @param reverse find the first or last pattern occurrence in error log
+   * @returns FileLocation or null
+   * examples: CMake Error at /home/test/tests.cmake:146 (message):
+   * examples: CMake Error at /home/test file name with spaces/tests.cmake:146:12 (message):
+   */
+  static getFileLocation(s: string, reverse: boolean): FileLocation | null {
+    let lines = s.trimStart().split("\n");
+    if (reverse) lines = lines.reverse();
+
+    // FIXME: this doesn't support file name which contains ':', but captures white space
+    const re = /(?:CMake Error at \/)?(?<file>[^:]+):(?<line>\d+)(:(?<col>\d+))?\b/;
```

Comment:
Maybe it's better to keep the `/` in `file` if exists. 
```suggestion
    const re = /(?:CMake Error at )?(?<file>[^:]+):(?<line>\d+)(:(?<col>\d+))?\b/;
```